lesson8/birds.rb

```diff
@@ -0,0 +1,39 @@
+argument0 = ARGV[0]
+argument1 = ARGV[1]
+
+if argument0.to_i >= 22 && argument0.to_i <= 30
+  abort 'Скорее идите в парк, соловьи поют!'
+
+elsif argument0.to_i >= 15 && argument0.to_i <= 35 && argument1.to_i == 1
+  abort 'Скорее идите в парк, соловьи поют!'
+
```

Comment:
we should deal with user input first and only then we can go with checking temperature and season.
```ruby
if temperature.nil?
  puts 'Какая сейчас температура?'
  temperature = STDIN.gets.chomp
end

if season.nil?
  puts 'Какое время года? (0 - весна, 1 - лето, 2 - осень, 3 - зима)?'
  season = STDIN.gets.chomp
end